mappings/net/minecraft/client/gui/hud/InGameHud.mapping

```diff
@@ -32,6 +32,7 @@ CLASS net/minecraft/class_329 net/minecraft/client/gui/hud/InGameHud
 	FIELD field_2041 overlayRemaining I
 	FIELD field_2042 ticks I
 	FIELD field_26527 DEMO_EXPIRED_MESSAGE Lnet/minecraft/class_2561;
+	FIELD field_26678 NAUSEA_TEXTURE Lnet/minecraft/class_2960;
```

Comment:
```suggestion
	FIELD field_26678 NAUSEA_TEXTURE_ID Lnet/minecraft/class_2960;
```

Reply:
This would be inconsistent with other names like "VIGNETTE_TEXTURE", "WIDGETS_TEXTURE" and "PUMPKIN_BLUR", which are IDs too and are used for similar purposes

Reply:
Nevermind